src/app/charts/row-chart/row-chart-view/row-chart-view.component.ts

```diff
@@ -235,12 +235,10 @@ export class RowChartViewComponent implements OnInit, OnDestroy, AfterViewInit,
                     }
                 })
                 .on('renderlet', function(chart) {
-                    if (self.firstRender) {
-                        self.firstRender = false;
-                        // Copy all vertical texts to another div, so they don't get hidden by
-                        // the row chart svg after being translated
-                        self.moveTextToElement(chart, self.stdCol.nativeElement, 9);
-                    }
+                    self.firstRender = false;
```

Comment:
I pulled down this branch and ran it locally, and I noticed that there is something going on with the x-axis label. See my comment on AG-419 for details. I'm not certain that it's due to the change you made here around self-firstRender, it's just a guess...